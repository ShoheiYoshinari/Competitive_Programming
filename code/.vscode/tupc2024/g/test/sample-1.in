3
4
1 -1 -2
3 3 -6
-1 2 -4
1 2 4
3
3 0 5
5 0 18
1 0 7
3
314 159 -1
313 158 -1000
315 160 999
