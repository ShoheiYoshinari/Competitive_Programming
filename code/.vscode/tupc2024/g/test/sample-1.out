72.0
0
0.0016129032
